presto-main/pom.xml

```diff
@@ -267,7 +267,7 @@
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <scope>test</scope>
+            <scope>provided</scope>
```

Comment:
Because the `com.facebook.presto.testing.assertions.Assert` class depends on TestNG's assertions, and it has to be in src/main so that other modules can use it